.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
Is this also grouped, like in firecracker?

Reply:
No, but I don't think this should be an issue, this only updates the rust-vmm-ci sub-module and not any of the dependencies.